cclient/serialization/json/json_serializer.c

```diff
@@ -1,116 +1,128 @@
-    const serializer* const s,
+    const serializer_t* const s,
     const find_transactions_req_t* const toSerialize) {
   return 0;
 }
 
 size_t json_find_transactions_deserialize_response_get_size(
-    const serializer* const s, const char* const toDesirialize) {
+    const serializer_t* const s, const char* const toDeserialize) {
   return 0;
 }
 
 // get_balances_response
-void json_get_balances_serialize_request(const serializer* const s,
+void json_get_balances_serialize_request(const serializer_t* const s,
                                          const get_balances_req_t* const obj,
                                          char* out) {}
 
-void json_get_balances_deserialize_response(const serializer* const s,
+void json_get_balances_deserialize_response(const serializer_t* const s,
                                             const char* const obj,
                                             get_balances_res_t* out) {}
 
 size_t json_get_balances_serialize_request_get_size_(
-    const serializer* const s, const get_balances_req_t* const toSerialize) {
+    const serializer_t* const s, const get_balances_req_t* const toSerialize) {
   return 0;
 }
 
 size_t json_get_balances_deserialize_response_get_size(
-    const serializer* const s, const char* const toDesirialize) {
+    const serializer_t* const s, const char* const toDeserialize) {
   return 0;
 }
 
 // get_inclusion_state_response
 void json_get_inclusion_state_serialize_request(
-    const serializer* const s, const get_inclusion_state_req_t* const obj,
+    const serializer_t* const s, const get_inclusion_state_req_t* const obj,
     char* out) {}
 
 void json_get_inclusion_state_deserialize_response(
-    const serializer* const s, const char* const obj,
+    const serializer_t* const s, const char* const obj,
     get_inclusion_state_res_t* out) {}
 
 size_t json_get_inclusion_state_serialize_request_get_size(
-    const serializer* const s,
+    const serializer_t* const s,
     const get_inclusion_state_req_t* const toSerialize) {
   return 0;
 }
 
 size_t json_get_inclusion_state_deserialize_response_get_size(
-    const serializer* const s, const char* const toDesirialize) {
+    const serializer_t* const s, const char* const toDeserialize) {
   return 0;
 }
 
 // get_neighbors_response
 
-void json_get_neighbors_deserialize_response(const serializer* const s,
+void json_get_neighbors_deserialize_response(const serializer_t* const s,
                                              const char* const obj,
                                              get_neighbors_res_t* out) {}
 
 size_t json_get_neighbors_deserialize_response_get_size(
-    const serializer* const s, const char* const toDesirialize) {
+    const serializer_t* const s, const char* const toDeserialize) {
   return 0;
 }
 
-// get_node_info_response
+// get_node_info
+
+size_t json_get_node_info_serialize_request_get_size(const serializer_t* const s){
+  //{"command": "getNodeInfo"}'
+  return 27;
```

Comment:
It could be more verbose if this line is rewritten as following:
```C
  return sizeof("{\"command\": \"getNodeInfo\"}");
```
Instead of hardcoded `27`.